src/pyk/kast.py

```diff
@@ -256,16 +256,27 @@ def ml_pred(self) -> KInner:
         return ml_term
 
 
+SORT_ATTRIBUTE: Final[str] = 'org.kframework.kore.Sort'
+
+
 @final
 @dataclass(frozen=True)
 class KVariable(KInner, WithKAtt):
     name: str
     att: KAtt
 
-    def __init__(self, name: str, att: KAtt = EMPTY_ATT):
+    def __init__(self, name: str, *, sort: Optional['KSort'] = None, att: KAtt = EMPTY_ATT):
```

Comment:
Consider moving `KVariable` after `KSort`, then

```suggestion
    def __init__(self, name: str, *, sort: Optional[KSort] = None, att: KAtt = EMPTY_ATT):
```

Reply:
Done.